api/envoy/config/common/matcher/v3/matcher.proto

```diff
@@ -2,7 +2,9 @@ syntax = "proto3";
 
 package envoy.config.common.matcher.v3;
 
+import "envoy/config/core/v3/extension.proto";
```

Comment:
I realize its merged, but one thing that could be good is to mark this as `work_in_progress` or `[#not-implement-yet:]` until we settle on implementation.

Reply:
Yeah good idea, will put a quick PR up